ioc/.htaccess

```diff
@@ -5,12 +5,24 @@ RewriteEngine on
 Options -MultiViews
 
 AddType text/turtle .ttl
+AddType application/rdf+xml .rdf
 
-# Rewrite rule to serve HTML content from the vocabulary URI if requested
-RewriteRule ^$ https://ip.pages.rwth-aachen.de/ioc/core/
-
-# If suffix ttl, redirect to turtle version
+# If accept header <text/turtle>
+RewriteCond %{HTTP_ACCEPT} ^.*text/turtle.* 
 RewriteRule ^$ https://ip.pages.rwth-aachen.de/ioc/core/ontology.ttl
 
+# If accept header <application/rdf+xml>
+RewriteCond %{HTTP_ACCEPT} application/rdf\+xml
+RewriteRule ^$ https://ip.pages.rwth-aachen.de/ioc/core/ontology.xml
+
+# If suffix ttl, redirect to turtle 
+RewriteRule ^.ttl$ https://ip.pages.rwth-aachen.de/ioc/core/ontology.ttl
```

Comment:
Do you want to literally redirect these extensions such as `https://w3id.org/ioc/.xyz`?